index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+
+<html>
+	
+	<head>
+		<meta charset="utf-8">
+		<title>Relaxr Blog</title>
+		<link rel="stylesheet" type="text/css" href="css/normalize.css">
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
+	</head>
+
+	 <body>
+			<header>
+				<div class="container">	
+					<h1 class="relaxr-logo">Relaxr</h1>
+					<nav class="main-nav">	
+						<ul>
+							<li class="nav-link">About</li>
+							<li class="nav-link">FAQ</li>
+							<li class="nav-link">Team</li>
+							<li class="nav-link">Contact Us</li>
+							<li class="nav-link">Blog</li>
+						</ul>
+					</nav>
+				</div>		
+			</header>
+			<div class="clearfix main">
+				<div class="container">
```

Comment:
The `clearfix` needs to go with the `container`, not the `main`. `clearfix` should always be applied to the immediate parent of the floated elements. In this case, you've floated the `<div class="content"></div>` and `<div class="sidebar"></div>`. Their immediate parent is the `<div class="container"></div>`, therefore its should be written as:
```
<div class="main">
  <div class="clearfix container">
    ....
  </div>
</div>
```